plugin/store/store_test.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 func TestFilterByCapNeg(t *testing.T) {
-	p := v2.NewPlugin("test", "1234567890", "/run/docker", "latest")
+	p := v2.NewPlugin("test", "1234567890", "/run/docker", "/var/run/docker/plugins/1234567890", "latest")
```

Comment:
Having all these as positional arguments does make it hard to grasp. Guess this was decided on already, just notice it now